test/helpers/mock-project.js

```diff
@@ -0,0 +1,42 @@
+var assert = require('assert');
+var fs = require('fs-extra');
+var spawnSync = require('child_process').spawnSync;
+
+var fixtureBower = require('../fixtures/bower.json');
+
+
+function MockProject(root) {
+  assert.equal(!!root, true, 'Must provide a root');
+  this.projectRoot = root;
+}
```

Comment:
The idea here is to create something like a project, so that ember-try can operate on it and then we can assert for the expected state

```
bower.json
bower_components/
```